src/store/actions/authAction.ts

```diff
@@ -0,0 +1,7 @@
+import actionTypes from '../actionTypes';
+
+export const setIsLoggedIn = (value: boolean) =>
+  ({
+    type: actionTypes.auth.LOGIN,
+    value,
```

Comment:
action creators should return objects with properties `type` & `payload`. Please change `value` to `payload: value`.